lib/kubernetes-deploy/restart_task.rb

```diff
@@ -45,6 +45,11 @@ def perform(deployments_names)
         raise ArgumentError, "#perform takes at least one deployment to restart"
       end
 
+      if deployments_names == "all"
```

Comment:
Shipit has no data about what deployments in the app so kubernetes-deploy would have to query the list of all deployments first.